com.unity.robotics.urdf-importer/Tests/Runtime/SensorTests.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using System.Collections.Generic;
+using NUnit.Framework;
+using UnityEngine;
+using UnityEngine.TestTools;
+using RosSharp.Urdf;
+using System.Xml;
+using System.Xml.Linq;
+
+public class SensorTests
+{
+    // A Test behaves as an ordinary method
+    [Test]
+    public void TotalDataTest()
+    {
+        int totalData = 13;
+        XDocument xdoc = XDocument.Load("Assets/test.txt");
+        Sensor sensor = new Sensor(xdoc.Element("sensor"));
+        Assert.AreEqual(totalData,sensor.elements.Count);
+    }
+
+    [Test]
+    public void NumberofAttributeTest()
+    {
+        int totalAttributes = 5;
+        int numberofAttributes = 0;
+        XDocument xdoc = XDocument.Load("Assets/test.txt");
```

Comment:
loading the Asset in the project will be problematic since it is outside the package. Either the test.txt has to be moved inside the package under Tests\  or you can have the content as a C# String inside the source code.